base/client.jl

```diff
@@ -668,3 +668,14 @@ macro main(args...)
         esc(:main)
     end
 end
+
+try
+    Base._start()  # or whatever the main function is
```

Comment:
I don't get this. Doesn't this run while loading client.jl, not when `main` is called? I think the place to add an exception handler is `_main` in juliac-buildscript.jl. And I don't think it has to be specific to MethodError; it can just do whatever the normal default exception handler does for all exceptions.